tor_ocr/main.py

```diff
@@ -107,18 +154,16 @@ def run(config):
         return
 
     try:
-        result = process_image(filename)
+        result = process_image(url)
     except RuntimeError:
```

Comment:
Is there anything within the `process_image()` method that we expect would raise a runtime error?

From context clues with the logged message below, it looks like this was something from when we were using the Tesseract library, where it might try to OCR an HTML page representing an Imgur album when we expected a direct link to the Imgur image.

Reply:
Shouldn't we be filtering only the specific extensions? I figured that we were, but no, nothing will be raised as of right now. I'll check the response we get when sending something like a website or similar later.

Reply:
@perryprog We don't filter for extensions, we filter for domains. This bot could receive any kind of link from i.redd.it or imgur.com.

Reply:
In that case I'll see what the result is, and that it should just return `None` and carry on.

Reply:
Is calls an exception now.

Reply:
I see now that we're raising the OCRError exception when there's a problem, but since it doesn't inherit from the RuntimeError type, this won't catch it.

Do we want to change the signature of the exceptions caught to `OCRError`, or maybe am I missing something and there is a RuntimeError that could be raised?

Reply:
Hmm... I think I'll change the Try-Catch to be a OCRError.